src/licensedcode/data/rules/free-unknown_112.RULE

```diff
@@ -2,6 +2,7 @@
 license_expression: free-unknown
 is_license_clue: yes
 relevance: 100
+is_deprecated: yes
```

Comment:
I get why you are deprecating this, but have you checked that there may be an opportunity to add few a common rules that use this as part of a larger license notice?
See https://github.com/search?q="OSI+Approved+License"&type=code

Reply:
Actually this was deprecated by mistake, reinstating this, as this is a clue allright. Will also look into the other instances linked here.